### PR TITLE
Fixing #3208

### PR DESCRIPTION
**Pull request by alexjeen**
https://github.com/Codeception/Codeception/issues/3208

### CODE REVIEW

**Comment by alexjeen** [comment]
I see that my Sublime added some unnecessary dots, it's only a fix in the loading of a folder and a piece of documentation

File: src/Codeception/Module/DataFactory.php [hunk 1]
```diff
@@ -163,7 +163,11 @@ public function _beforeSuite($settings = [])
 
         if ($this->config['factories']) {
             foreach ((array) $this->config['factories'] as $factoryPath) {
-                $this->factoryMuffin->loadFactories(realpath(codecept_root_dir() . $factoryPath));
+                $realpath = realpath(codecept_root_dir().$factoryPath);
+                if ($realpath === false) {
+                    throw new \Exception('The path to one of your factories is not correct. Please specify the directory relative to the codeception.yml file (ie. _support/factories).');
```
Comment (DavertMik) [inline src/Codeception/Module/DataFactory.php]: I think we can have ModuleException here. Anyway, it is easy to add it

**Comment by alexjeen** [comment]
Must I make new pull-request? I throw ModuleException now and cleaned up code to be PSR-2:

https://github.com/alexjeen/Codeception

**Comment by DavertMik** [comment]
Thanks!

**Pull request merged**